src/styles/style.css

```diff
@@ -0,0 +1,117 @@
+.body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  box-sizing: border-box;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__subtitle {
+  box-sizing: border-box;
+  color: #616070;
```

Comment:
Consider using the same color values as defined in the SCSS variables for consistency. For example, replace `#616070` with the variable `$text-color` equivalent.